src/bootstrap/setup.rs

```diff
@@ -40,9 +40,9 @@ pub fn setup(src_path: &Path, include_name: &str) {
     println!("`x.py` will now use the configuration at {}", include_path);
 
     let suggestions = match include_name {
-        "codegen" | "compiler" => &["check", "build", "test"][..],
+        "llvm" | "codegen" | "compiler" => &["check", "build", "test"][..],
         "library" => &["check", "build", "test library/std", "doc"],
-        "user" => &["dist", "build"],
+        "maintainer" | "user" => &["dist", "build"],
```

Comment:
Hmm, this was not introduced in this PR, but why does the top one produce a slice with `[..]` but the other two do not?

Reply:
Otherwise the compiler thinks these are references to arrays instead of slices: https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=df2c504584f4f824d5299a94d03eeac6